snapcraft.yaml

```diff
@@ -548,6 +548,20 @@ parts:
       sbin/: bin/
       usr/lib/: lib/
       usr/sbin/: bin/
+    override-build: |-
+      # Patch lvm.conf
```

Comment:
I'll merge this for now, but please can we expand this comment to explain what is happening here and why. Thanks